### DISCUSSION TITLE
MY game does even boot

### DISCUSSION THREAD

**Discussion by ToasterClock202** [Raytwo/ARCropolis#553] [Issues]
My game just doesnt load like not even black sceen it just leaves me at the ryujinx menu and this string of code caught my eye

00:00:08.882 |I| HLE.OsThread.43 ServiceSfdnsres GetAddrInfoRequestImpl: Trying to resolve: api.github.com

idk if that will help but thats all i got

**Comment by hikaruisasleep** [comment]
Disable guest network access
